reference/anbox-http-api.md

```diff
@@ -146,42 +146,15 @@ Input:
 
 ```json
 {
-    "latitude": 52.4538982,         # Latitude of geographic coordinates
+    "latitude": 52.4538982,          # Latitude of geographic coordinates
     "longitude": 13.3857982,         # Longitude of geographic coordinates
     "altitude": 10.0,                # Altitude in meters
     "time": 1597237057,              # Current time in millisecond since 1970-01-01 00:00:00 UTC
     "speed": 0.0,                    # Speed in meters per second
     "bearing": 0.0,                  # Magnetic heading in degrees
-    "format": "wgs84"                # (optional) Location format; possible values are "nmea" or "wgs84". Defaults to "wgs84" if not specified{
-  "metadata": {
-    "active_sensors": [
-      {
-        "delay": 66,
-        "type": "proximity"
-      },
-      {
-        "delay": 200,
-        "type": "acceleration"
-      }
-    ],
-    "enabled": true,
-    "supported_sensors": [
-      "humidity",
-      "pressure",
-      "light",
-      "proximity",
-      "temperature",
-      "orientation",
-      "magnetic-field",
-      "gyroscope",
-      "acceleration"
-    ]
-  },
-  "status": "Success",
-  "status_code": 200,
-  "type": "sync"
-}
-
+    "format": "wgs84",               # (optional) Location format; possible values are "nmea" or "wgs84". Defaults to "wgs84" if not specified
+    "horizontal_accuracy": 20,       # (optional) horizontal accuracy in meters. Defaults to 20 if not specified
+    "vertical_accuracy": 20          # (optional) vertical accuracy in meters. Defaults to 20 if not specified
```

Comment:
There's a note just above the code block that explains the format - can you add the explanation there instead of in a comment?

Reply:
Yes, I can. However, I feel it looks lengthy if we move the meaning of each field from the comment to the area which is wrapped by note syntax. Having them in comments looks more structured and easy to figure out what's the meaning of each field for readers. (Maybe I was wrong :))
How do you feel about this? 
Meanwhile, I just found in some places we are using **// <-** in the code block to explain the meaning of one field in the JSON output.  I need to fix that for consistency.

Reply:
I think it's okay to use comments to explain what a field contains (like "Latitude of geographic coordinates") - but when it comes to explaining different options and default values, it's hard to be clear enough in just a comment. And especially in this case where we have different options, and the code only shows one of them (namely wgs84).
So I suggest to keep the comments short and precise, and add additional information in the main docs.